auth/lib/utils/lock_screen_settings.dart

```diff
@@ -38,6 +41,23 @@ class LockScreenSettings {
 
     ///Workaround for privacyScreen not working when app is killed and opened.
     await setHideAppContent(getShouldHideAppContent());
+
+    await setHasMigratedLockScreenChanges();
+  }
+
+  Future<void> setHasMigratedLockScreenChanges() async {
+    if (_preferences.getBool(keyHasMigratedLockScreenChanges) == null ||
+        getIsAppLockSet() == false) {
+      await _preferences.setBool(keyHasMigratedLockScreenChanges, true);
```

Comment:
Move this to the end. So that only after everything is done we set this.
Why: let's say `isPinSet()` throws an error, the migration never completes but we set the key to true.